packages/liveblocks-core/src/client.ts

```diff
@@ -158,43 +154,19 @@ export type InternalSyncStatus = SyncStatus | "has-local-changes";
  * of Liveblocks, NEVER USE ANY OF THESE DIRECTLY, because bad things
  * will probably happen if you do.
  */
-export type PrivateClientApi<U extends BaseUserMeta, M extends BaseMetadata> = {
-  readonly currentUserIdStore: Store<string | null>;
+export type PrivateClientApi<U extends BaseUserMeta> = {
+  readonly currentUserIdStore: Store<string | undefined>;
   readonly mentionSuggestionsCache: Map<string, string[]>;
   readonly resolveMentionSuggestions: ClientOptions<U>["resolveMentionSuggestions"];
   readonly usersStore: BatchStore<U["info"] | undefined, string>;
   readonly roomsInfoStore: BatchStore<DRI | undefined, string>;
   readonly getRoomIds: () => string[];
-  readonly getUserThreads_experimental: (
-    options: GetUserThreadsOptions<M>
-  ) => Promise<{
-    threads: ThreadData<M>[];
-    inboxNotifications: InboxNotificationData[];
-    nextCursor: string | null;
-    requestedAt: Date;
-  }>;
-  readonly getUserThreadsSince_experimental: (
-    options: GetUserThreadsSinceOptions
-  ) => Promise<{
-    inboxNotifications: {
-      updated: InboxNotificationData[];
-      deleted: InboxNotificationDeleteInfo[];
-    };
-    threads: {
-      updated: ThreadData<M>[];
-      deleted: ThreadDeleteInfo[];
-    };
-    requestedAt: Date;
-  }>;
```

Comment:
All of these http methods are also part of return type of `createHttpClient` method, so these are removed in favour of the `httpClient` internal property, which also holds a bunch of other http methods like `getThreads({ roomId, ... }`, `getInboxNotifications({ roomId, ... })`